<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wgt_JobConvertImage</class>
 <widget class="QWidget" name="wgt_JobConvertImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>128</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <bold>false</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Convert image</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Convert image</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="QualitySL">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="QualityLabel">
       <property name="text">
        <string notr="true">90%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Quality :</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="RescalCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>QVGA (240 rows)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HVGA (320 rows)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VGA/WVGA (480 rows)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SD/DVD (576 rows)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>720p (720 rows)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XGA (768 rows)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1080p (1080 rows)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">5 MPix</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">6 MPix</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">8 MPix</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">10 MPix</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">12 MPix</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="RescalCB">
     <property name="text">
      <string>Resize :</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QCheckBox" name="DontUpscaleCB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>But not upscale image</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="DestFormatCB">
     <item>
      <property name="text">
       <string notr="true">jpg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">png</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Format :</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>DestFormatCB</tabstop>
  <tabstop>QualitySL</tabstop>
  <tabstop>RescalCB</tabstop>
  <tabstop>RescalCombo</tabstop>
  <tabstop>DontUpscaleCB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
